### PR TITLE
JavaScript Standard Style

### PR DESCRIPTION
**Pull request by feross**
https://github.com/feross/standard

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -10,7 +10,7 @@ var readTorrent = require('read-torrent')
 var proc = require('child_process')
 var peerflix = require('./')
 var keypress = require('keypress')
-var open = require('open')
+var openUrl = require('open')
```
Comment (mafintosh) [inline app.js]: was this needed because of standard? (just curious)

**Comment by mafintosh** [comment]
Nice. Added a travis hook as well.

**Comment by mafintosh** [comment]
Released in 0.25.1

File: app.js (hunk 1 above)
Comment (feross) [inline app.js]: Yeah, this is an ESLint bug, unfortunately. https://github.com/eslint/eslint/issues/892

It thinks we're overriding the global `window.open`, I think. :/ A workaround would be to remove the "no-undef" rule (detection of unused variables), but it's a really useful rule despite this bug...

**Pull request merged**